app/Model/AppModel.php

```diff
@@ -1195,7 +1195,76 @@ public function isPlugin($type = null) {
     
     return false;
   }
-  
+
+  /**
+   * This is copied by the framework itself but we take into consideration the soft delete functionality COmanage
+   * has in place.
+   *
+   * Returns false if any fields passed match any (by default, all if $or = false) of their matching values.
+   *
+   * Can be used as a validation method. When used as a validation method, the `$or` parameter
+   * contains an array of fields to be validated.
+   *
+   * @param array $fields Field/value pairs to search (if no values specified, they are pulled from $this->data)
+   * @param bool|array $or If false, all fields specified must match in order for a false return value
+   * @return bool False if any records matching any fields are found
+   */
+  public function isUniqueChangelog($fields, $or = true) {
```

Comment:
I'm not sure why we're adding this check? We support multiple copies of the same EmailAddress or Identifier on the same record, for example this is a common use case when syncing records from multiple Org Identity Sources.

Reply:
@boshrin let's discuss this over the Dev call.

Reply:
@boshrin Removed the `isUniqueChangelog` validation rule from the `EmailAddress` and the `Url` Model